packages/components/src/breadcrumbs/index.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { Breadcrumbs } from './';
+
+const meta: Meta< typeof Breadcrumbs > = {
+	title: 'packages/components/Breadcrumbs',
+	component: Breadcrumbs,
+	tags: [ 'autodocs' ],
+	parameters: {
+		actions: { argTypesRegex: '^on.*' },
+	},
+};
+
+export default meta;
+type Story = StoryObj< typeof meta >;
+
+export const Default: Story = {
+	args: {
+		items: [
+			{ label: 'Home', href: '#' },
```

Comment:
These `#` don't seem to be working nicely, they're getting the user out of Storybook into an iframe mode.

Should we use `javascript:void(0)` instead?